examples/tidb-ng-monitoring/tngm.yaml

```diff
@@ -0,0 +1,182 @@
+apiVersion: pingcap.com/v1alpha1
```

Comment:
What about moving this file to the `advanced` dir and only putting a basic example here?